apex/contrib/examples/gpu_direct_storage/example_load.py

```diff
@@ -0,0 +1,10 @@
+import torch
+import apex.contrib.gpu_direct_storage as gds
+
+torch.cuda.set_device(0)
+
+for size in [128, 1024, 8192]:
+    x = torch.empty(size, device = "cuda")
+    gds.load_data(x, f"{size}.data")
```

Comment:
How complex would we expect converting the `.data` produced by the current API to the `.pt` format that PyTorch typically uses (assuming that we are doing this in host-code)

Reply:
I'm imagining that it would look something like:
gds materializes the tensors on storage (fast),
host wraps the storage in `.pt` files (hopefully can be done without materializing the tensor in host memory)

Reply:
A `tensor.pt` is basically a `.zip` that has the following structure:
```
.
├── byteorder
├── .data
│   └── serialization_id
├── data
│   └── 0
├── data.pkl
└── version
```
In this structure `gds.load_data` and `gds.save_data` should should be responsible for operating on `data/0` file only. 

Looking from the higher-level prospective, the serializer should handle zipping/unzipping the `*.pt` file, write/read `byteorder`, `data.pkl` ..., and do `gds.load_data`/`gds.save_data` on `data/0`. I hope that `torch.serialization` does not materialize it :innocent: 

The main idea is to make it more atomic, so that other serializers like [safetensors](https://github.com/huggingface/safetensors) could use it.